docs/tutorials/azuredeploy.aci.simulation.json

```diff
@@ -152,27 +152,28 @@
         "instanceId": "[take(uniqueString(subscription().subscriptionId, resourceGroup().id, parameters('factoryName')), 7)]",
         "factoryCode": "[parameters('factoryName')]",
         "nameInfix": "[tolower(concat(variables('factoryCode'), '-', variables('instanceId')))]",
-        
```

Comment:
Please remove trailing whitespaces on line `155` and `162`.

Reply:
Those are trailing whitespaces that were removed (red background), not added (green background).

Reply:
Ah, sorry :D